events/bingo/bingoCardApp/bingoCard_app.js

```diff
@@ -17,7 +17,7 @@
     // START UTILITY FUNCTIONS
     // *************************************
 
-    /* ENCODE URL PARAMETERS: Formats data to send to Google sheet*/
+    // Takes a JSON object and translates that into URL query parameters
```

Comment:
Since Query Params always start with ?, maybe build that into this function so you don’t have to use concatenation later on

Reply:
because this function is used once in this file, the concatenation is only going to happen once no matter what. but if this function were used more than once, i'd totally agree with you and would change it.